pkg/roachpb/errors.go

```diff
@@ -950,17 +951,17 @@ func (e *ReadWithinUncertaintyIntervalError) SafeFormat(s redact.SafePrinter, _
 
 func (e *ReadWithinUncertaintyIntervalError) printError(p Printer) {
 	p.Printf("ReadWithinUncertaintyIntervalError: read at time %s encountered "+
-		"previous write with future timestamp %s within uncertainty interval `t <= "+
-		"(local=%v, global=%v)`; "+
+		"previous write with future timestamp %s (local=%s) within uncertainty interval `t <= "+
```

Comment:
Would it make sense to omit the `(local=%s)` when it's equal to `valueTs`? Seems like that would be the common case, in which case it can be a bit noisy. No strong opinion either way.

Reply:
Good idea! Done.